nodejs-rest-http-helm/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
It would be good to add to the README.md an overview of the chart files and what need to be changed/linked to what we have in the example